spec/controllers/redirect_controller_spec.rb

```diff
@@ -20,7 +16,7 @@
   describe "#item" do
     it "redirects to item page" do
       get :item, uuid: fedora3uuid1
-      expect(response).to redirect_to "http://test.host/files/#{gf.id}"
+      expect(response).to redirect_to generic_file_path(gf.id)
```

Comment:
I much rather be using the route helpers instead of hard coding these? (what happens if these route changes? now someone has to go update these tests)